### ISSUE TITLE
[2.42] FileNotFoundException

### ISSUE THREAD

**Issue by tchegito** [tchegito/zildo#132]
Seen on Google console:

(16 december)
```

java.lang.RuntimeException: 
  at com.alembrum.AndroidFileUtil.prepareSaveFile (AndroidFileUtil.java:109)
  at zildo.client.PlatformDependentPlugin.prepareSaveFile (PlatformDependentPlugin.java:142)
  at zildo.fwk.file.EasyWritingFile.saveFile (EasyWritingFile.java:49)
  at zildo.client.gui.menu.PlayerNameMenu.savePlayerName (PlayerNameMenu.java:81)
  at zildo.client.gui.menu.PlayerNameMenu.access$1 (PlayerNameMenu.java:77)
  at zildo.client.gui.menu.PlayerNameMenu$1.run (PlayerNameMenu.java:52)
  at zildo.client.Client.mainLoop (Client.java:242)
  at com.alembrum.OpenGLRenderer.onDrawFrame (OpenGLRenderer.java:107)
  at android.opengl.GLSurfaceView$GLThread.guardedRun (GLSurfaceView.java:1590)
  at android.opengl.GLSurfaceView$GLThread.run (GLSurfaceView.java:1286)
Caused by: java.io.FileNotFoundException: 
   at libcore.io.IoBridge.open (IoBridge.java:487)
   at java.io.FileOutputStream.<init> (FileOutputStream.java:87)
   at android.app.ContextImpl.openFileOutput (ContextImpl.java:410)
   at com.alembrum.AndroidFileUtil.prepareSaveFile (AndroidFileUtil.java:107)
Caused by: android.system.ErrnoException: 
   at libcore.io.Posix.open (Native Method)
   at libcore.io.BlockGuardOs.open (BlockGuardOs.java:186)
   at libcore.io.IoBridge.open (IoBridge.java:473)
```

**Comment by tchegito** [comment]
Maybe a "permission denied". Can happen in external storage (see here: https://github.com/square/picasso/issues/1361)

But it would be hard to understand, reproduce and fix !

**Comment by tchegito** [comment]
Fixed in 2.44

c119edc763d20a0123b1da544b4b14f643de0a33
